### PR TITLE
Descriptions changed for VWO Web Destination

### PR DESCRIPTION
**Pull request by prathamesh-88**
Simple Description changes for VWO Browser destination. Minor change in the utility file 

_A summary of your pull request, including the what change you're making and why._

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [X] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [X] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Comment by prathamesh-88** [comment]
Hi @joe-ayoub-segment 
No worries. I've reverted the changes for Cloud Destination. Will be opening a separate PR for the same.

File: packages/browser-destinations/src/destinations/vwo/utility.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 export function formatAttributes(attributes: { [k: string]: unknown }) {
   const formattedAttributes: { [k: string]: unknown } = {}
   for (const key in attributes) {
-    formattedAttributes[`segment_${key}`] = attributes[key]
+    formattedAttributes[`segment.${key}`] = attributes[key]
```
Comment (joe-ayoub-segment) [inline packages/browser-destinations/src/destinations/vwo/utility.ts:4]: hi @prathamesh-88 , this change will change the payload that gets sent to your platform when an identify() call comes to Segment. 

Is this going to be a problem for customers who are already using the vwo Integration? 
For example will it matter to a customer if a trait name changes from segment_first_name to segment.first_name? 

We need to be careful when making changes that change the data that customers are already collecting. Keen to hear your thoughts.

File: packages/browser-destinations/src/destinations/vwo/utility.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   }
   return formattedAttributes
 }
 
 export function sanitiseEventName(name: string) {
-  return 'segment_' + name
+  return 'segment.' + name
```
Comment (joe-ayoub-segment) [inline packages/browser-destinations/src/destinations/vwo/utility.ts:10]: Same comment as above.

**Review by joe-ayoub-segment — CHANGES_REQUESTED** [review]
Hi @prathamesh-88 , 

It all looks good apart from the change being made to trait names. I commented inline. Could you clarify if this change will result in a change to the data being collected by customers who are already using the Integration please?

**Comment by prathamesh-88** [comment]
Hi @joe-ayoub-segment , thanks for the prompt response.
This destination is in private beta and is being tested internally. It's not available to the customers.

**Review by joe-ayoub-segment — APPROVED** [review]
Looks good. Partner indicated no customers are using it yet.

**Pull request merged**